### PR TITLE
Fix documentation for UI_API

### PR DESCRIPTION
**Pull request by alaturqua**
## Description
- Fixed documentation to specify `web-ui.authentication.type` as `FORM` instead of `PASSWORD`.
- Ensures users configure the Trino Gateway correctly for the `ui/api/stats` REST endpoint.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
* 
```

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -328,7 +328,7 @@ Other timeout parameters are not applicable to the JDBC connection.
 
 This pulls cluster information from the `ui/api/stats` REST endpoint. This is
 supported for legacy reasons and may be deprecated in the future. It is only 
-supported for backend clusters with `web-ui.authentication.type = PASSWORD`. Set
+supported for backend clusters with `web-ui.authentication.type = FORM`. Set
```
Comment (mosabua) [inline docs/installation.md:331]:
```suggestion
supported for backend clusters with `web-ui.authentication.type=FORM`. Set
```

**Review by mosabua — APPROVED** [review]
Looks good .. please confirm @willmostly

File: docs/installation.md (hunk 1 above)
Comment (alaturqua) [reply docs/installation.md]: Took your change over and squashed commits.

**Comment by mosabua** [comment]
Please adjust commit message to shorter title. See https://cbea.ms/git-commit/

**Comment by alaturqua** [comment]
Done

**Review by willmostly — APPROVED** [review]
lgtm

**Review by mosabua — APPROVED** [review]

**Pull request merged**